### PR TITLE
New Mobile Links

### PR DESCRIPTION
**Pull request by pedrouid**

### CODE REVIEW

File: packages/helpers/qrcode-modal/src/browser/languages/fr.ts [hunk 1]
```diff
@@ -1,8 +1,17 @@
...
+import { TextMap } from "../types";
+
+const fr: TextMap = {
   choose_preferred_wallet: "Choisissez votre portefeuille préféré",
   connect_mobile_wallet: "Se connecter au portefeuille mobile",
   scan_qrcode_with_wallet: "Scannez le QR code avec un portefeuille compatible WalletConnect",
-  return_to_mobile_options: "Retour aux options du portefeuille mobile",
-  view_qrcode_option: "Voir le QR code",
   connect: "Se connecter",
+  // TODO: review following translations
```
Comment (pedrouid) [inline packages/helpers/qrcode-modal/src/browser/languages/fr.ts:8]: Could you take a look at the following translations? cc @PierrickGT

File: packages/helpers/qrcode-modal/src/browser/languages/fa.ts [hunk 2]
```diff
@@ -1,8 +1,17 @@
...
+import { TextMap } from "../types";
+
+const fa: TextMap = {
   choose_preferred_wallet: "کیف پول مورد نظر خود را انتخاب کنید",
   connect_mobile_wallet: "به کیف پول موبایل وصل شوید",
   scan_qrcode_with_wallet: "کد QR را با یک کیف پول سازگار با WalletConnect اسکن کنید",
-  return_to_mobile_options: "به گزینه های کیف پول موبایل برگردید",
-  view_qrcode_option: "مشاهده کد QR",
   connect: "اتصال",
+  // TODO: review following translations
```
Comment (pedrouid) [inline packages/helpers/qrcode-modal/src/browser/languages/fa.ts:8]: Could you take a look at the following translations? cc @shayanb

File: packages/helpers/qrcode-modal/src/browser/languages/de.ts [hunk 3]
```diff
@@ -1,8 +1,17 @@
...
+import { TextMap } from "../types";
+
+const de: TextMap = {
   choose_preferred_wallet: "Wähle bevorzugte Wallet",
   connect_mobile_wallet: "Verbinde mit Mobile Wallet",
   scan_qrcode_with_wallet: "Scanne den QR-code mit einer WalletConnect kompatiblen Wallet",
-  return_to_mobile_options: "Zurück zur Wallet-Anbieter Übersicht",
-  view_qrcode_option: "QR-code Anzeigen",
   connect: "Verbinden",
+  // TODO: review following translations
```
Comment (pedrouid) [inline packages/helpers/qrcode-modal/src/browser/languages/de.ts:8]: Could you take a look at the following translations? cc @MAkzent

File: packages/helpers/qrcode-modal/src/browser/languages/zh.ts [hunk 4]
```diff
@@ -1,8 +1,17 @@
...
+import { TextMap } from "../types";
+
+const zh: TextMap = {
   choose_preferred_wallet: "选择你的钱包",
   connect_mobile_wallet: "连接至移动端钱包",
   scan_qrcode_with_wallet: "使用兼容 WalletConnect 的钱包扫描二维码",
-  return_to_mobile_options: "回到移动端钱包选项",
-  view_qrcode_option: "查看二维码",
   connect: "连接",
+  // TODO: review following translations
```
Comment (pedrouid) [inline packages/helpers/qrcode-modal/src/browser/languages/zh.ts:8]: Could you take a look at the following translations? cc @p0s

File: packages/helpers/qrcode-modal/src/browser/languages/ko.ts [hunk 5]
```diff
@@ -1,8 +1,17 @@
...
+import { TextMap } from "../types";
+
+const ko: TextMap = {
   choose_preferred_wallet: "원하는 지갑을 선택하세요",
   connect_mobile_wallet: "모바일 지갑과 연결",
   scan_qrcode_with_wallet: "WalletConnect 지원 지갑에서 QR코드를 스캔하세요",
-  return_to_mobile_options: "모바일 지갑 옵션으로 돌아가기",
-  view_qrcode_option: "QR 코드 보기",
   connect: "연결",
+  // TODO: review following translations
```
Comment (pedrouid) [inline packages/helpers/qrcode-modal/src/browser/languages/ko.ts:8]: Could you take a look at the following translations? cc @paromix

File: packages/helpers/qrcode-modal/src/browser/languages/fa.ts (hunk 2 above)
Comment (shayanb) [reply packages/helpers/qrcode-modal/src/browser/languages/fa.ts:8]: It needs some changes, I'll push a PR for it.

File: packages/helpers/qrcode-modal/src/browser/languages/fr.ts (hunk 1 above)
Comment (PierrickGT) [reply packages/helpers/qrcode-modal/src/browser/languages/fr.ts:8]: @pedrouid looks good to me.

File: packages/helpers/qrcode-modal/src/browser/languages/de.ts (hunk 3 above)
Comment (MAkzent) [reply packages/helpers/qrcode-modal/src/browser/languages/de.ts:8]: looks good!

File: packages/helpers/qrcode-modal/src/browser/languages/fa.ts (hunk 2 above)
Comment (pedrouid) [reply packages/helpers/qrcode-modal/src/browser/languages/fa.ts:8]: Appreciate it! 🙏

**Comment by pedrouid** [comment]
Merging into next and releasing an rc

File: packages/helpers/qrcode-modal/src/browser/languages/zh.ts (hunk 4 above)
Comment (p0s) [reply packages/helpers/qrcode-modal/src/browser/languages/zh.ts:8]: Looks good!

I'd only change to this, which is clearer:

`   copied_to_clipboard: "复制到剪贴板成功！",`

File: packages/helpers/qrcode-modal/src/browser/languages/zh.ts [hunk 6]
```diff
@@ -1,8 +1,17 @@
...
+  qrcode: "二维码",
+  mobile: "移动",
+  copy_to_clipboard: "复制到剪贴板",
+  copied_to_clipboard: "复制到剪贴板！",
```
Comment (p0s) [inline packages/helpers/qrcode-modal/src/browser/languages/zh.ts:12]:
```suggestion
  copied_to_clipboard: "复制到剪贴板成功！",
```

Comment (pedrouid) [reply packages/helpers/qrcode-modal/src/browser/languages/zh.ts:12]: Thanks 👍 fixed on commit 15919c61d93fe434162fce9b22af26bceca7298c

**Pull request merged**